2023-09-02 14:11:28,987 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:29,602 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:32,116 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:32,181 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:32,397 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2396)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:35,149 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:35,522 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:35,922 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:11:59,930 - root - CRITICAL
__________________________________________________
	Expectation while wrapper
	Error type: <class 'httpx.ReadTimeout'>
	The read operation timed out
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:15:46,927 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:18:19,270 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:18:25,215 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:33:22,668 - root - CRITICAL
__________________________________________________
	Expectation while wrapper
	Error type: <class 'httpx.ReadTimeout'>
	The read operation timed out
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:36:55,363 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:36:56,840 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:36:57,231 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:36:58,269 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:01,897 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:03,441 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:03,458 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=how+has+the+world+changed+%3F+how+could+it+be+different+%3F (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:04,060 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:05,096 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:05,898 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=en&sl=vi&q=l%C3%A0m+th%E1%BA%BF+n%C3%A0o+th%E1%BA%BF+gi%E1%BB%9Bi+thay+%C4%91%E1%BB%95i+%3F+l%C3%A0m+th%E1%BA%BF+n%C3%A0o+n%C3%B3+c%C3%B3+th%E1%BB%83+kh%C3%A1c+%C4%91i+%3F (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:06,473 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:06,849 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:06,856 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:07,468 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:07,897 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:08,336 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:09,914 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:10,430 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:10,554 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:10,564 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:10,660 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:10,989 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=was+the+tale+told+well+%3F (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:11,838 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:11,871 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:12,836 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:13,461 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:13,708 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=en&sl=vi&q=c%C3%A2u+chuy%E1%BB%87n+k%E1%BB%83+c%C3%B3+hay+kh%C3%B4ng+%3F (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:15,996 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='cdn.reverso.net', port=443): Max retries exceeded with url: /trans/v2.11.0/main.js (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:16,113 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='cdn.reverso.net', port=443): Max retries exceeded with url: /trans/v2.11.0/main.js (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:16,274 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:16,374 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:16,424 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:16,996 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:17,475 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:17,486 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:18,249 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=what+would+they+do+under+the+same+conditions+%3F (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:18,610 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:18,878 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:18,901 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:19,033 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:19,368 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:19,511 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:19,548 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:19,582 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:19,949 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:20,093 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:20,377 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:20,624 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:21,096 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.translate.com', port=443): Max retries exceeded with url: /machine-translation (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:21,110 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:21,194 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=and+they+themselves+had+not+known+they+cared+. (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:21,217 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'translators.server.TranslatorError'>
	Unsupported from_language[en_GB] in ['en', 'vi_VN'].
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:21,222 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=en&sl=vi&q=h%E1%BB%8D+s%E1%BA%BD+l%C3%A0m+g%C3%AC+n%E1%BA%BFu+b%E1%BB%8B+%C4%91%E1%BA%B7t+d%C6%B0%E1%BB%9Bi+t%C3%ACnh+tr%E1%BA%A1ng+t%C6%B0%C6%A1ng+t%E1%BB%B1+%3F (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:21,400 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=en&sl=vi&q=v%C3%A0+ch%C3%BAng+kh%C3%B4ng+t%E1%BB%B1+bi%E1%BA%BFt+r%E1%BA%B1ng+ch%C3%BAng+quan+t%C3%A2m+. (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:21,712 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=was+there+a+hidden+message+%3F (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:22,764 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:23,214 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:23,524 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='web-api.itranslateapp.com', port=443): Max retries exceeded with url: /v3/texts/translate (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:37:25,502 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 14:38:11,283 - root - CRITICAL
__________________________________________________
	Expectation while wrapper
	Error type: <class 'httpx.ReadTimeout'>
	The read operation timed out
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:27,098 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2d698a60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:27,267 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2e691b10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:28,056 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2e692170>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:28,272 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2d694a30>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:29,150 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c69cd90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:36,323 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2e6a5000>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:36,851 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef149d5ba0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:43,828 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef149d1ea0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:53,246 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef149018d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:19:54,801 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2e6b94b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:20,381 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c808e20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:20,535 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c808550>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:21,053 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c5a7b50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:21,172 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c5a7b50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:21,834 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c844a60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:22,068 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c848cd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:53,039 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c7647c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:53,476 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c7689d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:57,789 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c75bca0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:20:59,331 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c75f1c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:01,153 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c77dae0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:01,723 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c73a080>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:11,535 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1f050310>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:13,649 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef31c7c700>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:16,853 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c785ea0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:17,548 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c789ed0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:48,168 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c3e7850>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:48,841 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c3ec7c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:51,985 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c3e5210>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:21:53,557 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c3e53f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:01,426 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c84cdc0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:02,006 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c84ce80>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:21,675 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c716b00>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:21,677 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c75b0a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:21,947 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c327df0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:21,992 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c321510>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:34,739 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c72b2b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:34,954 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c72b1c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:56,679 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1f0fda50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:57,120 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c82c700>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:57,212 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c828820>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:22:57,461 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c83c3d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:09,548 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c5a7e20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:09,943 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1f0fc2e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:10,823 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c838340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:11,730 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c8334c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:36,805 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c7348e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:37,128 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c774160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:39,924 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c3255a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:43,036 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c324a60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:47,923 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c7443a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:48,330 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c71fcd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:48,935 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2396)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:49,644 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c78c670>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:50,630 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c870df0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:51,507 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c7802b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:51,994 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c7738b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:52,070 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c78ca60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:52,779 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:52,867 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c71f9a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:53,489 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:54,252 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=en&sl=vi&q=ch%C3%BAng+t%C3%B4i+th%C3%ADch+x%C3%A2y+d%E1%BB%B1ng+m%E1%BB%8Di+th%E1%BB%A9+. (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:55,488 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=we+like+to+build+things+. (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:56,008 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:56,509 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:57,382 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:57,828 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:57,844 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='www.systran.net', port=443): Max retries exceeded with url: /translate/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:57,934 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:58,113 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c75faf0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:58,245 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:59,267 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2c3f4790>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:23:59,724 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:00,255 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lingvanex.com', port=443): Max retries exceeded with url: /demo/ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:01,300 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:01,527 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:02,714 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:04,251 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c774370>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:05,066 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:05,285 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:05,554 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:05,736 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=we+genetically+engineer+bacteria+. (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:05,870 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: /m?tl=vi&sl=en&q=we+make+things+grow+. (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:07,808 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:07,825 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='lwt.languagewire.com', port=443): Max retries exceeded with url: /f/api/v1/auth/cookie (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:08,840 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1c75f7f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:09,061 - root - CRITICAL
__________________________________________________
	Expectation while deepGoogle
	Error type: <class 'requests.exceptions.ConnectionError'>
	('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:24:10,324 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.SSLError'>
	HTTPSConnectionPool(host='translate.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:997)')))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:25:33,448 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef1f035db0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:25:33,739 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2eb31cc0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:25:40,286 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2eb2f610>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:25:40,341 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2eb2ef80>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:25:54,407 - root - CRITICAL
__________________________________________________
	Expectation while wrapper
	Error type: <class 'httpx.ReadTimeout'>
	The read operation timed out
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:26:00,484 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2eb10c70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:26:00,623 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef2eb08910>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:26:05,543 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef31c69ab0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
2023-09-02 16:26:05,796 - root - CRITICAL
__________________________________________________
	Expectation while translatorsTransSub
	Error type: <class 'requests.exceptions.ConnectionError'>
	HTTPConnectionPool(host='mt.utibet.edu.cn', port=80): Max retries exceeded with url: /mt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fef31c69a20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
